source/common/options.js

```diff
@@ -149,12 +149,23 @@ function buildOptionsPage() {
 
   settings.forEach(function (setting) {
     if (setting.type === 'checkbox') {
-      jq('#' + setting.section + 'SettingsPage').append(jq('<div>', { class: 'row option-row' }).append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' })).append(jq('<div>', { class: 'option-description' }).append(jq('<label>', { for: setting.name, text: setting.title })).append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', text: setting.description }))));
+      jq('#' + setting.section + 'SettingsPage')
+        .append(jq('<div>', { class: 'row option-row' })
+          .append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' }))
+          .append(jq('<div>', { class: 'option-description' })
+            .append(jq('<label>', { for: setting.name, text: setting.title }))
+            .append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block' }))));
+      jq('#' + setting.name + 'HelpBlock').html(setting.description);
```

Comment:
In which file would reallyLongString be placed?

I'll also take a look at js-xss.